cli/lib/otel/index.ts

```diff
@@ -1,15 +1,19 @@
-import { CollectorTraceExporter, CollectorMetricExporter } from '@opentelemetry/exporter-collector'
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger'
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus'
 import { HttpInstrumentation } from '@opentelemetry/instrumentation-http'
 import { WinstonInstrumentation } from '@opentelemetry/instrumentation-winston'
-import { Resource, envDetector, processDetector } from '@opentelemetry/resources'
+import { Resource } from '@opentelemetry/resources'
 import { NodeSDK } from '@opentelemetry/sdk-node'
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
 
+const jaegerExporter = new JaegerExporter()
+const prometheusExporter = new PrometheusExporter()
+
 const sdk = new NodeSDK({
   // Automatic detection is disabled, see comment below
   autoDetectResources: false,
-  traceExporter: new CollectorTraceExporter(),
-  metricExporter: new CollectorMetricExporter(),
+  traceExporter: jaegerExporter,
+  metricReader: prometheusExporter,
```

Comment:
The exporter is automatically loaded depending on the `OTLP_TRACES_EXPORTER` environment variable if the `traceExporter` here is omitted.

Unfortunately, it's not the case for the metrics exporter, and in our infra, we want to push metrics via OTLP, so

```suggestion
  metricReader: otlpMetricExporter,
```